### PR TITLE
jarek/5816: add load magic command

### PR DESCRIPTION
**Pull request by jaroslawmalekcodete**

### CODE REVIEW

File: kernel/demoProjects/loadMagicJarDemo/build.gradle [hunk 1]
```diff
@@ -0,0 +1,35 @@
+version 'unspecified'
```
Comment (scottdraves) [inline kernel/demoProjects/loadMagicJarDemo/build.gradle:16]: banners on all files

**Comment by scottdraves** [comment]
👍

File: doc/contents/LoadMagicCommand.ipynb [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+   "outputs": [],
+   "source": [
+    "%classpath add jar resources/jar/loadMagicJarDemo.jar"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Source code of ShowEvnsCustomMagicCommand class\n",
```
Comment (scottdraves) [inline doc/contents/LoadMagicCommand.ipynb:16]: link to the file on github instead of copying it here

File: kernel/demoProjects/build.gradle [hunk 3]
```diff
@@ -0,0 +1,20 @@
+version 'unspecified'
```
Comment (scottdraves) [inline kernel/demoProjects/build.gradle:16]: yea this is good and also please make the other demo jars work this way.

File: kernel/demoProjects/loadMagicJarDemo/src/main/java/com/twosigma/beakerx/custom/magic/command/ShowEvnsCustomMagicCommand.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+  @Override
+  public String getMagicCommandName() {
+    return "%showEnvs";
+  }
+
+  @Override
+  public MagicCommandOutcomeItem execute(MagicCommandExecutionParam param) {
+    MIMEContainer result = Text(System.getenv());
+    return new MagicCommandOutput(MagicCommandOutcomeItem.Status.OK,result.getData().toString());
```
Comment (scottdraves) [inline kernel/demoProjects/loadMagicJarDemo/src/main/java/com/twosigma/beakerx/custom/magic/command/ShowEvnsCustomMagicCommand.java:35]: > result.getData().toString()

can this part be simplified and given a name that makes sense?

File: kernel/demoProjects/build.gradle (hunk 3 above)
Comment (jaroslawmalekcodete) [reply kernel/demoProjects/build.gradle:16]: I added tablesawDemo project

File: kernel/demoProjects/loadMagicJarDemo/build.gradle (hunk 1 above)
Comment (jaroslawmalekcodete) [reply kernel/demoProjects/loadMagicJarDemo/build.gradle:16]: done

File: doc/contents/LoadMagicCommand.ipynb (hunk 2 above)
Comment (jaroslawmalekcodete) [reply doc/contents/LoadMagicCommand.ipynb:16]: We will do it after merging branch to the master, because no such file in master.

**Pull request merged**